src/main/java/uk/gov/companieshouse/acsp/manage/users/controller/UserAcspMembership.java

```diff
@@ -22,13 +25,15 @@
 @RestController
 public class UserAcspMembership implements UserAcspMembershipInterface {
 
-  private final AcspMembersService acspMembersService;
-
+  public static final String PLEASE_CHECK_THE_REQUEST_AND_TRY_AGAIN = "Please check the request and try again";
   private static final Logger LOG =
       LoggerFactory.getLogger(StaticPropertyUtil.APPLICATION_NAMESPACE);
+  private final AcspMembersService acspMembersService;
+  private final AcspMembershipService acspMembershipService;
 
-  public UserAcspMembership(final AcspMembersService acspMembersService) {
+  public UserAcspMembership(final AcspMembersService acspMembersService, AcspMembershipService acspMembershipService) {
```

Comment:
Can we make this final to be consistent?

```suggestion
  public UserAcspMembership(final AcspMembersService acspMembersService, final AcspMembershipService acspMembershipService) {
```